site/dat/repo/various.json

```diff
@@ -2099,5 +2099,29 @@
         "downloadUrl": "https://github.com/Qytera-Gmbh/JMeterHARImporterPlugin/releases/download/v0.2.3/jmeter-har-import-plugin-v0.2.3.jar"
       }
     }
+  },
+  {
+    "id": "JmeterNamesTemplater",
+    "name": "JMeter Names Templater",
+    "description": "Jmeter Names Templater is a plugin for Jmeter that allows you to set naming templates for elements and rename them based on a described configuration file.",
+    "vendor": "Daniil Golikov",
+    "markerClass": "com.example.jmeter.plugin.gui.RenameElementsMenuCreator",
+    "helpUrl": "https://github.com/DanilGolikov/JmeterNamesTemplater/blob/master/README.md",
+    "componentClasses": [
+      "com.example.jmeter.plugin.gui.RenameElementsMenuCreator",
```

Comment:
`com.example.jmeter.plugin` does not look like a good namespace for plugin. Can you come up with something more distinctive? Something more author-specific.
Should be a simple refactoring.